eslint-remote-tester.config.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+const fs = require("node:fs");
+
+/** @type {import('eslint-remote-tester').Config} */
+module.exports = {
+    /** Repositories to scan */
+    repositories: [
+        // A few dozen top repositories using QUnit or this plugin.
+        "DevExpress/DevExtreme",
+        "adopted-ember-addons/ember-data-model-fragments",
+        "balanced/balanced-dashboard",
+        "ember-intl/ember-intl",
+        "emberjs/ember.js",
+        "getsentry/sentry-javascript",
+        "glimmerjs/glimmer-vm",
+        "hashicorp/boundary-ui",
+        "hotwired/stimulus",
+        "jashkenas/backbone",
+        "jquery/jquery",
+        "js-cookie/js-cookie",
+        "l10n-tw/canvas-lms",
+        "rust-lang/crates.io",
+        "simonihmig/ember-responsive-image",
+        "videojs/video.js"
+    ],
+
+    /** Extensions of files under scanning */
+    extensions: ["js", "mjs", "cjs", "ts", "mts", "cts"],
+
+    /** Optional boolean flag used to enable caching of cloned repositories. For CIs it's ideal to disable caching. Defaults to true. */
+    cache: false,
+
+    /** ESLint configuration */
+    eslintrc: {
+        plugins: ["qunit"],
+
+        // Enable all of our rules.
+        rules: Object.fromEntries(
+            fs
+                .readdirSync(`${__dirname}/lib/rules`)
+                .map((filename) => `qunit/${filename.replace(/\.js$/, "")}`)
```

Comment:
To future-proof this (if we switch to .ts, .mjs, or .mts), could this be made more generic? (E.g., by using `node:path` to strip the extension)

Reply:
Good point, updated to use `node:path` utilities.